samples/CookieSample/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.IO;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+
+namespace CookieSample
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            var config = new ConfigurationBuilder().AddEnvironmentVariables("ASPNETCORE_").Build();
```

Comment:
@glennc this is similar to our Docker discussion. Compare with the launchSettings.json.